compiler/x/codegen/OMRTreeEvaluator.cpp

```diff
@@ -1075,6 +1075,12 @@ TR::Register *OMR::X86::TreeEvaluator::integerStoreEvaluator(TR::Node *node, TR:
    if (instr && node->getOpCode().isIndirect())
       cg->setImplicitExceptionPoint(instr);
 
+   if (node->getSymbol()->isUnsafeShadowSymbol())
+      {
+      instr->setNeedsGCMap(0xFF00FFFF);
```

Comment:
Do you actually need registers in your GC map? Wherever this exception ends up being caught, whether in the same method or elsewhere, I don't think that the currently live registers will be live at the destination so there's no point in having them in the map, unless this is an x codegen specific quirk that I'm not aware of.